### DISCUSSION TITLE
Scene in IFRAME works only in Chrome incognito mode

### DISCUSSION THREAD

**Discussion by buddy90210** [google/model-viewer#2745] [Q&A]
When viewing a scene in an iframe, the browser crashes, but if viewed in incognito mode, everything works.
Tell me what could be wrong?

**Comment by elalish** [comment]
As always, there's no way to help unless you provide a repro link and repro steps.

**Reply by buddy90210** [reply]
https://dvatc.ru/frame

**Reply by buddy90210** [reply]
Mobile Chrome 92.0.4515.159
On Chrome desktop via WebXR emulator it works correctly.

**Reply by elalish** [reply]
I still need repro steps. Are you saying 3D mode is crashing or AR, or both? Also, I notice on your link there are several red errors in your console. Have you looked into those?
